modules/virt-template-fields-for-boot-source.adoc

```diff
@@ -14,21 +15,21 @@ The following table describes the fields for *Add boot source to template* windo
 |Upload local file (creates PVC)
 |Upload a file from your local device. Supported file types include gz, xz, tar, and qcow2.
 
-|Import via URL (creates PVC)
-|Import content from an image available from an HTTP or HTTPS endpoint. Obtain the download link URL from the web page where the image download is available and enter that URL link in the *Import via URL (creates PVC)* field. Example: For a Red Hat Enterprise Linux image, log on to the Red Hat Customer Portal, access the image download page, and copy the download link URL for the KVM guest image.
+|URL (creates PVC)
+|Import content from an image available from an HTTP or HTTPS endpoint. Obtain the download link URL from the web page where the image download is available and enter that URL link in the *Import URL* field. Example: For a Red Hat Enterprise Linux image, log on to the Red Hat Customer Portal, access the image download page, and copy the download link URL for the KVM guest image.
 
-|Clone existing PVC (creates PVC)
+|PVC (creates PVC)
 |Use a PVC that is already available in the cluster and clone it.
 
-|Import via Registry (creates PVC)
+|Registry (creates PVC)
 |Specify the bootable operating system container that is located in a registry and accessible from the cluster. Example: kubevirt/cirros-registry-dis-demo.
 
 |Source provider
 |
 |Optional field. Add descriptive text about the source for the template or the name of the user who created the template. Example: Red Hat.
 
-.3+|Advanced
-|Storage class
+.3+|Advanced Storage settings
```

Comment:
Is this referring to a UI element or should storage be lowercase?

Reply:
It's a UI element with poor capitalization. That's what the UI text says. If I had time, I would submit a bug.